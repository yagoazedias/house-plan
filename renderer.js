// This file is required by the index.html file and will
// be executed in the renderer process for that window.
// All of the Node.js APIs are available in this process.

import React from 'react'
import ReactDOM from 'react-dom'

class App extends React.Component {
  render () {
    return (
      <h1>Testing this will be something</h1>
    )
  }
}

ReactDOM.render(<App/>,document.getElementById('app'))
