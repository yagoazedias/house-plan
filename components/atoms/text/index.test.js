// Link.react.test.js
import React from 'react'
import renderer from 'react-test-renderer'
import { Text } from './index'

test('Testing text snapshot', () => {
    const component = renderer.create(<Text>My Test</Text>)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})