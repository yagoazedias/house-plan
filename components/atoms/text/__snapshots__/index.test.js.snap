// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing text snapshot 1`] = `
<span>
  My Test
</span>
`;
